### ISSUE TITLE
Can't start server

### ISSUE THREAD

**Issue by ryaniskira** [Xpra-org/xpra#3073]
OS: Archlinux, using xpra-git from the AUR which builds from source
Graphics: Nvidia

Starting today I can't seem to get xpra to start a server at all. `xpra start :10` will spit out no errors, but then `xpra list` shows no displays.  Reading the log there's not really anything obvious other than xpra not actually starting:

```
X.Org X Server 1.20.10
X Protocol Version 11, Revision 0
Build Operating System: Linux Arch Linux
Current Operating System: Linux Remilia-Scarlet 5.11.10-142-tkg-MuQSS #1 TKG SMP PREEMPT Mon, 29 Mar 2021 19:07:49 +0000 x86_64
Kernel command line: intel_pstate=passive BOOT_IMAGE=/vmlinuz-linux-tkg-muqss-haswell root=UUID=b5160ca7-9e35-4de3-b3d9-7d256e08a377 rw sysrq_always_enabled=1 nowatchdog systemd.unified_cgroup_hierarchy=1
Build Date: 14 December 2020  12:10:29PM
 
Current version of pixman: 0.40.0
	Before reporting problems, check http://wiki.x.org
	to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
	(++) from command line, (!!) notice, (II) informational,
	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/run/user/1000/xpra/Xorg.:10.log", Time: Mon Apr  5 20:18:45 2021
(++) Using config file: "/etc/xpra/xorg.conf"
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
```
I know there's supposed to be log info after that last line, but that's all I get, and the odd thing is if I attempt to start the display again the log displays nothing. It creates a new logfile `:10.log.new` but as mentioned it's empty. I am really at a loss because it is failing totally silently so there's nothing obvious going on.

**Comment by znmeb** [comment]
Anything interesting in `journalctl -r`? If the NVIDIA GPU didn't like something there would be something in there, I think.

**Comment by ryaniskira** [comment]
I didn't find anything from Nvidia, but I did find this
```
Apr 05 20:36:14 Remilia-Scarlet systemd[3484]: run-r9d823d745f3847f18ebf1ae72f77eda4.scope: Deactivated successfully.
Apr 05 20:36:14 Remilia-Scarlet systemd[3484]: Started xpra-start.
```
It seems like xpra is immediately dying? Or that could be completely normal to see the scope deactivate.

**Comment by znmeb** [comment]
I've got an NVIDIA laptop running Arch - I've been meaning to put Xpra on it so all my machines have it. If I can reproduce this I'll file a bug on AUR.

**Comment by znmeb** [comment]
It looks like the server is started and running as a daemon on my system:

```
$ xpra start :10
using systemd-run to wrap 'start' server command
Running scope as unit: run-rab51e0a06fa34c89b54c0e5963e9ba6f.scope
Entering daemon mode; any further errors will be reported to:
  /run/user/1000/xpra/:10.log
znmeb@DreamCatcher:~$ more '/run/user/1000/xpra/:10.log'
2021-04-05 18:20:15,434 cannot access python uinput module:
2021-04-05 18:20:15,435  No module named 'uinput'

X.Org X Server 1.20.10
X Protocol Version 11, Revision 0
Build Operating System: Linux Arch Linux
Current Operating System: Linux DreamCatcher 5.10.25-1-lts #1 SMP Sat, 20 Mar 2021 19:37:44 +0000 x86_64
Kernel command line: BOOT_IMAGE=/boot/vmlinuz-linux-lts root=UUID=62a5b49e-a6e9-4276-959a-fcfe3f93f850 rw quiet loglevel=3 audit=0
Build Date: 14 December 2020  12:10:29PM

Current version of pixman: 0.40.0
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/run/user/1000/xpra/Xorg.:10.log", Time: Mon Apr  5 18:20:15 2021
(++) Using config file: "/etc/xpra/xorg.conf"
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
$ ps -ef | grep xpra
znmeb      54347       1  0 18:20 ?        00:00:00 /usr/bin/python3.9 /usr/bin/xpra start :10 --systemd-run=no
znmeb      54348   54347  8 18:20 ?        00:00:02 /usr/lib/Xorg-for-Xpra-:10 -noreset -novtswitch -nolisten tcp +extension GLX +extension RANDR +extension RENDER -auth /home/znmeb/.Xauthority -logfile /run/user/1000/xpra/Xorg.:10.log -configdir /run/user/1000/xpra/xorg.conf.d/54347 -config /etc/xpra/xorg.conf -depth 24 :10
```

Try a `ps -ef | grep xpra`

**Comment by ryaniskira** [comment]
```
kira        4993       1  0 19:45 ?        00:00:04 /usr/lib/Xorg -noreset -novtswitch -nolisten tcp +extension GLX +extension RANDR +extension RENDER -auth /home/kira/.Xauthority -logfile /run/user/1000/xpra/Xorg.:10.log -configdir /run/user/1000/xpra/xorg.conf.d/4991 -config /etc/xpra/xorg.conf -depth 24 :10
kira       13128       1  0 20:18 ?        00:00:03 /usr/lib/Xorg-for-Xpra-:99 -noreset -novtswitch -nolisten tcp +extension GLX +extension RANDR +extension RENDER -auth /home/kira/.Xauthority -logfile /run/user/1000/xpra/Xorg.:99.log -configdir /run/user/1000/xpra/xorg.conf.d/13127 -config /etc/xpra/xorg.conf -depth 24 :99
kira       13649       1  0 20:23 ?        00:00:03 /usr/lib/Xorg-for-Xpra-:69 -noreset -novtswitch -nolisten tcp +extension GLX +extension RANDR +extension RENDER -auth /home/kira/.Xauthority -logfile /run/user/1000/xpra/Xorg.:69.log -configdir /run/user/1000/xpra/xorg.conf.d/13647 -config /etc/xpra/xorg.conf -depth 24 :69
kira       21791   21468  0 21:28 pts/7    00:00:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn --exclude-dir=.idea --exclude-dir=.tox xpra
```

**Comment by znmeb** [comment]
so it's running ... I just fired up `xpra start --start=firefox` and connected to it from another system so that's working.

**Comment by ryaniskira** [comment]
No dice for me.

**Comment by totaam** [comment]
@ryaniskira try to start with `--no-daemon`.
You may also want to add `--start-via-proxy=no` and `--systemd-run=no`.

**Comment by ryaniskira** [comment]
No dice, `xpra list` still shows no screens and I am unable to connect.

**Comment by totaam** [comment]
@ryaniskira at least you should see how far the server goes before it dies, which may tell us why it dies.
My guess is some problem with `Xorg`, if that's the case then you can switch to `Xvfb` in `/etc/xpra/conf.d/55_server_x11.conf`.

**Comment by ryaniskira** [comment]
@totaam Output stops at the same spot the log in the OP stops, and even as that's running doing `xpra list` in another terminal session shows no sessions found. Switching to Xvfb in the config is no dice either, `xpra list` still shows no screens running and running it with --no-daemon there's zero output.

**Comment by ryaniskira** [comment]
Update, it seems to run just OK in tcp mode: `xpra start --start=xterm --bind-tcp=0.0.0.0:10000` so it seem just the default mode (which I assume is SSH) is the issue.

**Comment by totaam** [comment]
> Output stops at the same spot the log in the OP stops

I have never seen that happen.
Is there still an `xpra` process running then?

> and running it with `--no-daemon` there's zero output

That doesn't make any sense.

> Update, it seems to run just OK in tcp mode

Again, that makes zero sense.
Adding a `bind-tcp` option to a server `start` command line changes nothing, only adds an extra socket. That cannot make it work better than before.

**Comment by ryaniskira** [comment]
> Is there still an `xpra` process running then?

According to ps, yes

> That doesn't make any sense.
> Again, that makes zero sense.
> Adding a bind-tcp option to a server start command line changes nothing, only adds an extra socket. That cannot make it work better than before.

I'm just as confused as you are, unless something is broken with my SSH setup but that would make no sense because I can SSH into the PC just fine.

**Comment by totaam** [comment]
> unless something is broken with my SSH setup

No. SSH is not involved here at all. You are not creating any SSH sockets.

**Comment by ryaniskira** [comment]
I mean I'm at a total loss for this situation either, but somehow creating a TCP socket to listen on works just fine:

```
xpra list                                                
Found the following xpra sessions:
/run/user/1000/xpra:
	LIVE session at :0
/home/kira/.xpra:
	LIVE session at :0
```

![Screenshot at 2021-04-06 11-07-14](https://user-images.githubusercontent.com/5086636/113733206-40387700-96c8-11eb-924b-f2a5445883e5.png)

**Comment by ryaniskira** [comment]
Update: Ok so I tried it in default mode without creating a TCP socket but also without specifying a display number, and that works too. So specifying a display seems to cause the issue?

**Comment by totaam** [comment]
> So specifying a display seems to cause the issue?

AFAICT, if that's the case, you have not posted a single example of failing command line.

**Comment by ryaniskira** [comment]
> AFAICT, if that's the case, you have not posted a single example of failing command line.

Because whatever is happening it's failing silently. `xpra start --no-daemon :10` produces:

```                             
using systemd-run to wrap 'start' server command
Running scope as unit: run-rc9ebc80bbdd8491bbcc086240a2c9dd1.scope
```
There's no fireworks which is what I'd expect if something is failing, just the noticeable lack of output. The only indication of failure is `xpra list` produces:

```
xpra list                 
Found the following xpra sessions:
/run/user/1000/xpra:
	LIVE session at :0
/home/kira/.xpra:
	LIVE session at :0
```
Which :0 is the TCP session, :10 is missing (this is the screen I attempted to create), and of course any client will fail to connect to :10:

```
...
server socket for display :10 not found
 waiting up to 20 seconds
xpra initialization error:
 cannot find live server for display :10
2021-04-06 11:20:38,086 Error: failed to receive anything, not an xpra server?
2021-04-06 11:20:38,086   could also be the wrong protocol, username, password or port
2021-04-06 11:20:38,086   or the session was not found
2021-04-06 11:20:38,086 Connection lost
```
I can already exclude possibilities of a wrong username, password, or port because I am using ssh config with a keyfile which this same config file will grant me ssh access just fine.

Just for reference, this is the output of that same command at the top of this comment sans the display number being specified, `xpra start --no-daemon`:

```
using systemd-run to wrap 'start' server command
Running scope as unit: run-rab2b95d92e864680ae4b563fce67200c.scope
2021-04-06 11:23:01,264 cannot use uinput for virtual devices:
2021-04-06 11:23:01,264  [Errno 13] Failed to open the uinput device: Permission denied
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running

X.Org X Server 1.20.10
X Protocol Version 11, Revision 0
Build Operating System: Linux Arch Linux
Current Operating System: Linux Patchouli-Knowledge 5.10.19-1032.native #1 SMP Wed Mar 3 13:57:49 PST 2021 x86_64
Kernel command line: initrd=\intel-ucode.cpio initrd=\initramfs-linux-clear-bin.img root=PARTUUID=88d1674e-35f8-4c94-adae-10e48685124e rootfstype=f2fs add_efi_memmap module.sig_unenforce initcall_debug intel_iommu=igfx_off kvm-intel.nested=1 no_timer_check noreplace-smp page_alloc.shuffle=1 rcupdate.rcu_expedited=1 console=tty0 console=ttyS0,115200n8 cryptomgr.notests tsc=reliable systemd.unified_cgroup_hierarchy=false
Build Date: 14 December 2020  12:10:29PM
 
Current version of pixman: 0.40.0
	Before reporting problems, check http://wiki.x.org
	to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
	(++) from command line, (!!) notice, (II) informational,
	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/run/user/1000/xpra/Xorg.S177510.log", Time: Tue Apr  6 11:23:01 2021
(++) Using config file: "/etc/xpra/xorg.conf"
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
2021-04-06 11:23:05,758 created unix domain socket '/run/user/1000/xpra/Patchouli-Knowledge-1'
2021-04-06 11:23:05,758 cannot create group socket '/run/xpra/Patchouli-Knowledge-1'
2021-04-06 11:23:05,758  [Errno 13] Permission denied
2021-04-06 11:23:05,759 created unix domain socket '/home/kira/.xpra/Patchouli-Knowledge-1'
2021-04-06 11:23:05,854 pointer device emulation using XTest
2021-04-06 11:23:05,924 Error: cannot find the html web root
2021-04-06 11:23:05,924  '/usr/share/xpra/www' does not exist
2021-04-06 11:23:06,834 Warning: failed to load the mdns publisher
2021-04-06 11:23:06,834  No module named 'zeroconf'
2021-04-06 11:23:06,834  either install the 'python-avahi' module
2021-04-06 11:23:06,834  or use the 'mdns=no' option
2021-04-06 11:23:06,834 Error setting up server dbus instance:
2021-04-06 11:23:06,834  No module named 'dbus'
2021-04-06 11:23:06,836 Warning: cannot forward notifications,
2021-04-06 11:23:06,836  failed to load or register our dbus notifications forwarder:
2021-04-06 11:23:06,836  No module named 'dbus'
2021-04-06 11:23:06,836  if you do not have a dedicated dbus session for this xpra instance,
2021-04-06 11:23:06,836  use the 'notifications=no' option
2021-04-06 11:23:06,844 Warning: cannot load dbus helper:
2021-04-06 11:23:06,844  No module named 'dbus'
2021-04-06 11:23:06,845 pulseaudio server started with pid 177656
2021-04-06 11:23:06,845  private server socket path:
2021-04-06 11:23:06,845  '/run/user/1000/xpra/pulse-1/pulse/native'

(Xpra:177510): Gtk-CRITICAL **: 11:23:06.908: gtk_widget_realize: assertion 'widget->priv->anchored || GTK_IS_INVISIBLE (widget)' failed
2021-04-06 11:23:06,913 Error: failed to import the virtual video module:
2021-04-06 11:23:06,913  No module named '_dbus_bindings'
2021-04-06 11:23:06,914 Warning: cannot watch for application menu changes without pyinotify:
2021-04-06 11:23:06,914  No module named 'pyinotify'
2021-04-06 11:23:06,977 Error parsing xdg menu data:
2021-04-06 11:23:06,977  ParsingError in file '/etc/xdg/menus/kde-applications.menu', File not found
2021-04-06 11:23:06,978  this is either a bug in python-xdg,
2021-04-06 11:23:06,981  or an invalid system menu configuration
2021-04-06 11:23:07,130 xpra is ready.
2021-04-06 11:23:07,130 xpra GTK3 X11 version 4.2 64-bit
2021-04-06 11:23:07,158 59.6GB of system memory
2021-04-06 11:23:07,171  uid=1000 (kira), gid=1000 (kira)
2021-04-06 11:23:07,171  running with pid 177510 on Linux unknown unknown unknown
2021-04-06 11:23:07,171  connected to X11 display :1 with 24 bit colors
2021-04-06 11:23:07,181 GStreamer version 1.18.4 for Python 3.9.2 64-bit
2021-04-06 11:23:08,817 Warning: OpenGL support check failed:
2021-04-06 11:23:08,817  unknown error
```

Yeah there's some fireworks due to some missing Python modules, but it works and clients can connect to it.

**Comment by totaam** [comment]
> Because whatever is happening it's failing silently. xpra start --no-daemon :10 produces:
> ```
> using systemd-run to wrap 'start' server command
> Running scope as unit: run-rc9ebc80bbdd8491bbcc086240a2c9dd1.scope
> ```

Try with `--systemd-run=no`.
And to get *all* the debug you will ever need, add `-d all`.

> I can already exclude possibilities of a wrong username, password, or port because I am using ssh config with a keyfile which this same config file will grant me ssh access just fine.

Like I said above. You are not using SSH anywhere in these examples.  
`xpra start` and `xpra list` have nothing to do with SSH.

**Comment by MaierOli2010** [comment]
I do have the same problem.  Specifying the display will lead to no session found. 
I've tried with:
 `xpra start :501 --systemd-run=no -d all `
results in no running session.
The .X501 lock file is created as well as a socket in `/tmp/.X11-unix/`
Also the process seems to be running and connected to the socket `lsof -U:`
```
    xpra      19209 omaier    3u  unix 0x00000000ee929711      0t0 63691 type=STREAM (CONNECTED)
    Xvfb      19211 omaier    4u  unix 0x00000000f72498e8      0t0 60094 @/tmp/.X11-unix/X501 type=STREAM (LISTEN)\
    Xvfb      19211 omaier    5u  unix 0x00000000eedf931f      0t0 60095 /tmp/.X11-unix/X501 type=STREAM (LISTEN)
````
Processes are running and can be found using `htop`.

Problem started to occur after a system update yesterday. Running Arch-Linux with latest updates and s NVIDIA GPU. Previously everything worked with xpra 3.X

Output of the successful start without display number:
```

2021-04-08 11:50:36,521 cannot use uinput for virtual devices:^[[0m
2021-04-08 11:50:36,521  [Errno 19] Failed to open the uinput device: No such device
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
2021-04-08 11:50:37,788 created unix domain socket '/run/user/1000/xpra/fbmt129-2'
2021-04-08 11:50:37,788 created unix domain socket '/run/xpra/fbmt129-2'
2021-04-08 11:50:37,788 created unix domain socket '/home/omaier/.xpra/fbmt129-2'
2021-04-08 11:50:37,827 pointer device emulation using XTest
2021-04-08 11:50:38,055 Error: cannot find the html web root
2021-04-08 11:50:38,055  '/usr/share/xpra/www' does not exist
2021-04-08 11:50:38,211 Warning: failed to load the mdns publisher
2021-04-08 11:50:38,211  No module named 'zeroconf'
2021-04-08 11:50:38,211  either install the 'python-avahi' module
2021-04-08 11:50:38,211  or use the 'mdns=no' option
2021-04-08 11:50:38,218 D-Bus notification forwarding is available
2021-04-08 11:50:38,225 pulseaudio server started with pid 19293
2021-04-08 11:50:38,225  private server socket path:
2021-04-08 11:50:38,225  '/run/user/1000/xpra/pulse-2/pulse/native'

(Xpra:19274): Gtk-CRITICAL **: 11:50:38.274: gtk_widget_realize: assertion 'widget->priv->anchored || GTK_IS_INVISIBLE (widget)' failed
2021-04-08 11:50:38,278 Warning: webcam forwarding is disabled
2021-04-08 11:50:38,278  the virtual video directory '/sys/devices/virtual/video4linux' was not found
2021-04-08 11:50:38,278  make sure that the 'v4l2loopback' kernel module is installed and loaded
2021-04-08 11:50:38,278  or use the 'webcam=no' option
2021-04-08 11:50:38,278 found 0 virtual video devices for webcam forwarding
2021-04-08 11:50:38,286 printer forwarding enabled using postscript and pdf
2021-04-08 11:50:38,286 Warning: cannot watch for application menu changes without pyinotify:
2021-04-08 11:50:38,286  No module named 'pyinotify'
2021-04-08 11:50:38,288 Warning: cannot use application menu data:
2021-04-08 11:50:38,288  no python-xdg module
2021-04-08 11:50:38,356 62.7GB of system memory
2021-04-08 11:50:38,438 xpra is ready.
2021-04-08 11:50:38,439 xpra GTK3 X11 version 4.2 64-bit
2021-04-08 11:50:38,462 GStreamer version 1.18.4 for Python 3.9.2 64-bit
2021-04-08 11:50:38,481  uid=1000 (omaier), gid=1000 (omaier)
2021-04-08 11:50:38,481  running with pid 19274 on Linux Arch rolling n/a
2021-04-08 11:50:38,481  connected to X11 display :2 with 24 bit colors
2021-04-08 11:50:39,494 Warning: OpenGL support check failed:
2021-04-08 11:50:39,494  unknown error
```


And for a failed start with specified display:
```

2021-04-08 11:37:16,260 setting up tcp sockets:
2021-04-08 11:37:16,260 setting up ssl sockets: 
2021-04-08 11:37:16,260 setting up ssh sockets: 
2021-04-08 11:37:16,260 setting up ws sockets: 
2021-04-08 11:37:16,260 setting up wss sockets: 
2021-04-08 11:37:16,260 setting up rfb sockets: 
2021-04-08 11:37:16,260 setting up UDP sockets: 
2021-04-08 11:37:16,261 setting up vsock sockets: 
2021-04-08 11:37:16,261 sd_listen_fds(0)=0
2021-04-08 11:37:16,261 get_sd_listen_sockets()=[]
2021-04-08 11:37:16,261 systemd sockets: []
2021-04-08 11:37:16,261 env=environ({'USER': 'omaier', 'LOGNAME': 'omaier', 'HOME': '/home/omaier', 'PATH': '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games', 'SHELL': '/bin/zsh', 'TERM': 'xterm', 'MOTD_SHOWN': 'pam', 'MAIL': '/var/spool/mail/omaier', 'XDG_RUNTIME_DIR': '/run/user/1000>
2021-04-08 11:37:17,266 verify_display_ready(None, ':501', False)
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/xpra/x11/vfb_util.py", line 431, in verify_display_ready
    wait_for_x_server(strtobytes(display_name), timeout)
  File "xpra/x11/bindings/wait_for_x_server.pyx", line 41, in xpra.x11.bindings.wait_for_x_server.wait_for_x_server
RuntimeError: could not connect to X server on display ':501' after 1 seconds
2021-04-08 11:37:17,272 has_uinput()
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/xpra/server/server_util.py", line 334, in has_uinput
    uinput.fdopen()         #@UndefinedVariable
  File "/usr/lib/python3.9/site-packages/uinput/__init__.py", line 84, in fdopen
    return _libsuinput.suinput_open()
  File "/usr/lib/python3.9/site-packages/uinput/__init__.py", line 70, in _open_error_handler
    raise OSError(code, msg)
OSError: [Errno 19] Failed to open the uinput device: No such device
2021-04-08 11:37:17,272 cannot use uinput for virtual devices:
2021-04-08 11:37:17,272  [Errno 19] Failed to open the uinput device: No such device
2021-04-08 11:37:17,272 start_Xvfb('Xvfb +extension GLX +extension Composite -screen 0  8192x4096x24+32 -nolisten tcp -noreset -auth $XAUTHORITY  -dpi 96', '', 24, ':501', '/home/omaier/Downloads/xpra-git', 1000, 1000, 'omaier', '23925edd987c446da2ae5b2a5c8bb910', None)
2021-04-08 11:37:17,272 found existing XAUTHORITY file '/home/omaier/.Xauthority'
2021-04-08 11:37:17,273 xvfb_cmd=['Xvfb-for-Xpra-:501', '+extension', 'GLX', '+extension', 'Composite', '-screen', '0', '8192x4096x24+32', '-nolisten', 'tcp', '-noreset', '-auth', '/home/omaier/.Xauthority', '-dpi', '96', ':501']
2021-04-08 11:37:17,277 xvfb process=<Popen: returncode: None args: ['Xvfb-for-Xpra-:501', '+extension', 'GLX', '...>
2021-04-08 11:37:17,277 display_name=:501
2021-04-08 11:37:18,283 dbus_launch='dbus-launch --sh-syntax --close-stderr', current DBUS_SESSION_BUS_ADDRESS=None
2021-04-08 11:37:18,284 start_dbus(dbus-launch --sh-syntax --close-stderr) env={'USER': 'omaier', 'LOGNAME': 'omaier', 'HOME': '/home/omaier', 'PATH': '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games', 'SHELL': '/bin/zsh', 'XDG_RUNTIME_DIR': '/run/user/1000', 'SSH_CLIENT': '129.27.171.>
2021-04-08 11:37:18,290 out(['dbus-launch', '--sh-syntax', '--close-stderr'])=b"DBUS_SESSION_BUS_ADDRESS='unix:abstract=/tmp/dbus-PPZfD3OHLz,guid=70d409c33043e01bb7f01952606ecece';\nexport DBUS_SESSION_BUS_ADDRESS;\nDBUS_SESSION_BUS_PID=19217;\n"
2021-04-08 11:37:18,290 dbus_pid=19217, dbus-env={'DBUS_SESSION_BUS_ADDRESS': 'unix:abstract=/tmp/dbus-PPZfD3OHLz,guid=70d409c33043e01bb7f01952606ecece', 'DBUS_SESSION_BUS_PID': '19217'}
```


I've noticed the following error in the Xorg log files which might be relevant:
```
[    52.183] (II) xfree86: Adding drm device (/dev/dri/card0)
[    52.184] (EE) systemd-logind: failed to take device /dev/dri/card0: Operation not permitted
```

FYI connecting to the started service works over network as well.

I hope this helps. Going to test if the problem persists after a downgrade to Xpra 3.X now.

**Comment by totaam** [comment]
I believe this was fixed in 4.1.3, please re-open with more details if not.

ie the output of:
```shell
XPRA_ALL_DEBUG=1 xpra start :10
```
